src/main/java/org/broadinstitute/hellbender/tools/reference/CompareReferences.java

```diff
@@ -93,6 +105,15 @@ public class CompareReferences extends GATKTool {
     @Argument(fullName = "display-only-differing-sequences", doc = "If provided, only display sequence names that differ in their actual sequence.", optional = true)
     private boolean onlyDisplayDifferingSequences = false;
 
+    @Argument(fullName = "base-comparison", doc = "If provided, any mismatching, same-length sequences will be aligned for a base-comparison.", optional = true)
```

Comment:
Argument description is incorrect, since full alignment mode can handle sequences of different lengths. Should be something like: `Mode for base-level comparisons. Off by default, but can do either full alignment of mismatching sequences to find both SNPs and indels (FULL_ALIGNMENT), or can find SNPs only in mismatching sequences of the same length (FIND_SNPS_ONLY)`